crates/feldera-types/src/format/avro.rs

```diff
@@ -54,3 +60,60 @@ pub struct AvroEncoderConfig {
     /// password-based authentication (see `registry_username` and `registry_password`).
     pub registry_authorization_token: Option<String>,
 }
+
+/// Avro output format configuration.
+#[derive(Clone, Serialize, Deserialize, Debug, Default, ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct AvroParserConfig {
+    /// Format used to encode data change events in this stream.
+    ///
+    /// The default value is 'raw'.
+    #[serde(default)]
+    pub update_format: AvroUpdateFormat,
+
+    /// Avro schema used to encode output records.
+    ///
+    /// Specified as a string containing schema definition in JSON format.
+    /// This schema must match precisely the SQL view definition, including
+    /// nullability of columns.
+    pub schema: Option<String>,
```

Comment:
why is this an Option?

Reply:
The normal behavior is to extract schema id from message headers and download the schema from the registry, rather than take it as part of the config.

Reply:
This probably should be documented